takhojin/models/userDao.js

```diff
@@ -1,16 +1,34 @@
 const dataSource = require("./dataSource");
 
-const createUser = async (email, password, description, profile_img) => {
+const createUser = async (email, password, description, profileImg) => {
   try {
-    return await dataSource.query(
+    await dataSource.query(
       `INSERT INTO users (
           email,
           password,
           description,
           profile_image
         ) VALUES (?,?,?,?);
         `,
-      [email, password, description, profile_img]
+      [email, password, description, profileImg]
+    );
+  } catch (err) {
+    console.log(err);
+    const error = new Error(`INVALID_DATA_INPUT`);
+    error.statusCode = 500;
+    throw error;
+  }
+};
+
+const login = async (email) => {
```

Comment:
```suggestion
const getUserByEmail = async (email) => {
```
dao는 Data Access Object의 약자로서 해당 부분에 작성되는 함수이름으로 login은 적합하지 않습니다. CRUD(create, get, update, delete)와 자원의 조합으로 getUserByEmail과 같은 식으로 작성해주세요!